app/src/main/java/de/westnordost/streetcomplete/osm/cycleway/CyclewayCreator.kt

```diff
@@ -49,6 +52,37 @@ fun LeftAndRightCycleway.applyTo(tags: Tags, isLeftHandTraffic: Boolean) {
     ).applyTo(tags)
 }
 
+/* bare cycleway tags are interpreted differently for oneways */
+private fun expandBareTags(tags: Tags, isLeftHandTraffic: Boolean) {
+    val cycleway = tags["cycleway"] ?: return
+    // i.e. they are only expanded into one side. Which side depends on country, direction of oneway
+    // and whether it is an "opposite" tag value
+    val side = if (isOneway(tags)) {
+        val isReverseSideRight = isReversedOneway(tags) xor isLeftHandTraffic
+        val isOpposite = cycleway.startsWith("opposite")
+        if (isOpposite == isReverseSideRight) "right" else "left"
+    } else {
+        "both"
+    }
+    if (!tags.containsKey("cycleway:$side")) {
+        tags["cycleway:$side"] = cycleway
+            .removePrefix("opposite_") // opposite_track -> track etc.
+            .replaceFirst("opposite", "no") // opposite -> no
+    }
+    tags.remove("cycleway")
+    tags.expandBareTagIntoSide("cycleway", "lane", side)
+    tags.expandBareTagIntoSide("cycleway", "lane", side)
```

Comment:
This line is duplicate, and the second line will do nothing. Did you mean `tags.expandBareTagIntoSide("cycleway", "", side)`?

Reply:
Whoops, thank you!